crates/ra_assists/Cargo.toml

```diff
@@ -12,6 +12,8 @@ itertools = "0.8.0"
 arrayvec = "0.4.10"
 rustc-hash = "1.0.1"
 rustc_lexer = "0.1.0"
+text_unit = "0.1.6"
+smol_str = "0.1.12"
```

Comment:
smol_str and text_unit are re-exported in `ra-syntax`, so there's no need to modify Cargo.toml

Reply:
It fails to build here if a remove these lines.

Reply:
I mean, you should 

```
use ra_sytax::{SmolStr, TextUnit, TextRange}
```

Reply:
Oh, I see now.